typegraph/tests/typegraphs/types.ts

```diff
@@ -0,0 +1,34 @@
+import { t, typegraph } from "../../deno/src/mod.ts";
+import { DenoRuntime } from "../../deno/src/runtimes/deno.ts";
+
+const a = t.integer();
+
+const s1 = t.struct({ a, b: t.integer({ min: 12 }) });
+
+const b = t.integer({ min: 12, max: 43 });
+
+const user = t.struct({
+  id: t.integer(),
+  post: t.proxy("Post"),
+}, { name: "User" });
+
+const post = t.struct({
+  id: t.integer(),
+  author: user,
+}, { name: "Post" });
+
+typegraph("test-types", (g) => {
+  g.expose({
+    one: DenoRuntime.func(s1, b, {
```

Comment:
I believe even though there is only one DenoRuntime, we shall stick with the same declaration process as others:

```
deno = DenoRuntime() # always return singleton
deno.func()
```